cmake/coverage.cmake

```diff
@@ -9,13 +9,16 @@ ov_coverage_clean(REPOSITORY "openvino"
 ov_coverage_capture(INFO_FILE "openvino"
                     BASE_DIRECTORY "${OV_COVERAGE_BASE_DIRECTORY}"
                     DIRECTORY "${OV_COVERAGE_GCDA_DATA_DIRECTORY}"
-                    EXCLUDE_PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/*.pb.cc" 
-                                     "${OV_COVERAGE_BASE_DIRECTORY}/*.pb.h" 
-                                     "${OV_COVERAGE_BASE_DIRECTORY}/*/tests/*" 
-                                     "${OV_COVERAGE_BASE_DIRECTORY}/*/tests_deprecated/*" 
+                    EXCLUDE_PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/*.pb.cc"
+                                     "${OV_COVERAGE_BASE_DIRECTORY}/*.pb.h"
+                                     "${OV_COVERAGE_BASE_DIRECTORY}/*/tests/*"
+                                     "${OV_COVERAGE_BASE_DIRECTORY}/*/tests_deprecated/*"
                                      "${OV_COVERAGE_BASE_DIRECTORY}/thirdparty/*") # Skip some pb files, tests and thirdparty
 # Common report
-ov_coverage_genhtml(INFO_FILE "openvino"
+ov_coverage_extract(INPUT "openvino" OUTPUT "openvino_all"
+                    PATTERNS "${OV_COVERAGE_BASE_DIRECTORY}/*")
+
+ov_coverage_genhtml(INFO_FILE "openvino_all"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
```

Comment:
The idea was to avoid additional targets. Users should use `ov_coverage target and got the full report.
I would like to use openvino target instead of introducing the new one